randovania/data/json_data/prime1.txt

```diff
@@ -2403,582 +2411,1261 @@ Ruins Entrance
 
 > Dock to Ruined Shrine Access; Heals? False
-  * Normal Door to Ruined Shrine Access/Dock to Main Plaza
+  * Missile Blast Shield to Ruined Shrine Access/Dock to Main Plaza
+  > Dock to Ruins Entrance
+      Trivial
 
 > Dock to Nursery Access; Heals? False
-  * Normal Door to Nursery Access/Dock to Main Plaza
+  * Morph Ball Door to Nursery Access/Dock to Main Plaza
+  > Dock to Ruins Entrance
+      Trivial
 
 > Dock to Plaza Access; Heals? False
-  * Normal Door to Plaza Access/Dock to Main Plaza
+  * Permanently Locked to Plaza Access/Dock to Main Plaza
+  > Dock to Ruins Entrance
+      Trivial
+  > Pickup (Energy Tank)
+      Trivial
 
 > Dock to Piston Tunnel; Heals? False
   * Normal Door to Piston Tunnel/Dock to Main Plaza
+  > Dock to Ruins Entrance
+      Trivial
+  > Pickup (Missile Expansion 3)
+      Grapple Beam
+
+> Pickup (Energy Tank); Heals? True
+  * Pickup 3; Major Location? False
+  > Dock to Plaza Access
+      Trivial
+
+> Pickup (Missile Expansion); Heals? False
+  * Pickup 0; Major Location? False
+  > Dock to Ruins Entrance
+      Trivial
+
+> Pickup (Missile Expansion 2); Heals? False
+  * Pickup 1; Major Location? False
+  > Dock to Ruins Entrance
+      Trivial
+
+> Pickup (Missile Expansion 3); Heals? False
+  * Pickup 2; Major Location? False
+  > Dock to Ruins Entrance
+      Trivial
 
 ----------------
 Ruined Fountain Access
 Asset id: 1443741240
 > Dock to Main Plaza; Heals? False
   * Normal Door to Main Plaza/Dock to Ruined Fountain Access
+  > Dock to Ruined Fountain
+      Morph Ball
 
 > Dock to Ruined Fountain; Heals? False
   * Normal Door to Ruined Fountain/Dock to Ruined Fountain Access
+  > Dock to Main Plaza
+      Morph Ball
 
 ----------------
 Ruined Shrine Access
 Asset id: 3748948704
 > Dock to Ruined Shrine; Heals? False
   * Normal Door to Ruined Shrine/Dock to Ruined Shrine Access
+  > Dock to Main Plaza
+      Trivial
 
 > Dock to Main Plaza; Heals? False
-  * Normal Door to Main Plaza/Dock to Ruined Shrine Access
+  * Missile Blast Shield to Main Plaza/Dock to Ruined Shrine Access
+  > Dock to Ruined Shrine
+      Trivial
 
 ----------------
 Nursery Access
 Asset id: 153979389
 > Dock to Eyon Tunnel; Heals? False
   * Normal Door to Eyon Tunnel/Dock to Nursery Access
+  > Dock to Main Plaza
+      Trivial
 
 > Dock to Main Plaza; Heals? False
   * Normal Door to Main Plaza/Dock to Nursery Access
+  > Dock to Eyon Tunnel
+      Trivial
 
 ----------------
 Plaza Access
 Asset id: 1396020311
 > Dock to Main Plaza; Heals? False
   * Normal Door to Main Plaza/Dock to Plaza Access
+  > Dock to Vault
+      Trivial
 
 > Dock to Vault; Heals? False
   * Normal Door to Vault/Dock to Plaza Access
+  > Dock to Main Plaza
+      Trivial
 
 ----------------
 Piston Tunnel
 Asset id: 725556462
 > Dock to Main Plaza; Heals? False
-  * Normal Door to Main Plaza/Dock to Piston Tunnel
+  * Morph Ball Door to Main Plaza/Dock to Piston Tunnel
+  > Dock to Training Chamber
+      Trivial
```

Comment:
Add morph requirement

Reply:
isn't it implied by setting both docks to morph ball door?

Reply:
Done